grouper/fe/templates/permission-group.html

```diff
@@ -0,0 +1,176 @@
+{% extends "base.html" %}
+{% from 'macros/ui.html' import audit_log_panel, paginator, dropdown with context %}
+
+{% block subtitle %} permission {{ permission.name }}{% endblock %}
+
+{% block heading %}
+Permissions
+{% endblock %}
+
+{% block subheading %}
+{{ permission.name }} {{total}} permission(s)
```

Comment:
I think `permission(s)` -> `grant(s)` since that's what the page is displaying now. (Same for the permission-service-account template below.)